app/uk/gov/hmrc/tai/viewModels/IncomeSourceViewModel.scala

```diff
@@ -35,11 +35,12 @@ case class IncomeSourceViewModel(
   displayEndDate: Boolean,
   detailsLinkLabel: String,
   detailsLinkUrl: String,
+  taxCodeUrl: String = "",
```

Comment:
Is a string the best data type to represent this. I would prefer a Call over a string and if it has the option of not being required maybe a `Option[Call]` would be suitable